addons/website/views/snippets/s_bento_banner.xml

```diff
@@ -0,0 +1,52 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+<template id="s_bento_banner" name="Bento Banner">
+    <section class="s_bento_banner pt32 pb32">
+        <div class="container">
+            <section
+                data-name="Card"
+                class="s_col_no_resize py-5 py-lg-0 d-flex o_cc o_cc5 oe_img_bg o_bg_img_center overflow-hidden rounded-4 shadow-lg"
```

Comment:
I would add `oe_unmovable oe_unremovable` to avoid the user to break the layout by mistake : 
https://www.awesomescreenshot.com/video/39856928?key=3c314c859b2bd56c253cff68ca72adc5

Also, don't forget to reorder the classes (eg. `d-flex` before padding classes :) )